amoro-ams/amoro-ams-server/pom.xml

```diff
@@ -472,4 +472,38 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>paimon</id>
```

Comment:
`support-paimon-format`,`support-hudi-format`,`support-all-formats` may be proper names.